src/gr11review/part2/Utility.java

```diff
@@ -1,5 +1,327 @@
 package gr11review.part2;
 
+import java.io.*;
+import java.util.*;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+
+
+/** 
+* Method 2: Add numbers together
+* @author: H. Chung
+*
+*/
 public class Utility {
-    
-}
\ No newline at end of file
+  public static int sumNumbers(String str) {
+    // Setting Variables
+    int intLength;
+    intLength = str.length();
+    int intCount;
+    int intSum = 0;
+    String strWord = "";
+    // Looping until every character is check through
+    for (intCount = 0; intCount < intLength; intCount++) {
+      // See if the value is a digit
+      if (Character.isDigit(str.charAt(intCount))) {
+        // Conditions to see if you can add a number in the string
+        if (intCount < intLength-1 && Character.isDigit(str.charAt(intCount+1))) {
+           // Finding a number within the sentence
+          strWord += str.charAt(intCount);
+        } else {
+          // Adding the numbers together
+          strWord += str.charAt(intCount);
+          intSum += Integer.parseInt(strWord);
+          strWord = "";
+        }
+      }
+    }
+    // return sum
+    return intSum;
+  }
+  /** 
+  * File IO 2: Reading words in file and picking the first alphabetically first word
+  * @author: H. Chung
+  *
+  */
+  public static String alphaWord (String filenametxt) throws IOException {
+    BufferedReader FileText = new BufferedReader(new FileReader(filenametxt));
+    // Setting Variables and Reading File
+    String strWord;
+    String strFinalWord;
+    int intCompare;
+    strWord = FileText.readLine();
+    strFinalWord = FileText.readLine();
```

Comment:
@Haydenchung03 method fails when there is only one word in the text file.
```
Cannot invoke "String.compareToIgnoreCase(String)" because "strFinalWord" is null
	at gr11review.part2.Utility.alphaWord(Utility.java:57)
```